lib/thrift/muzic/include/defl_static.h

```diff
@@ -30,8 +30,8 @@
    They may be altered/distinct from the originals used in PuTTY source
    code. */
 
-void outbits(struct uzlib_comp *ctx, unsigned long bits, int nbits);
-void zlib_start_block(struct uzlib_comp *ctx);
-void zlib_finish_block(struct uzlib_comp *ctx);
-void zlib_literal(struct uzlib_comp *ctx, unsigned char c);
-void zlib_match(struct uzlib_comp *ctx, int distance, int len);
+void outbits(struct uzlib_comp* ctx, unsigned long bits, int nbits);
+void zlib_start_block(struct uzlib_comp* ctx);
+void zlib_finish_block(struct uzlib_comp* ctx);
+void zlib_literal(struct uzlib_comp* ctx, unsigned char c);
+void zlib_match(struct uzlib_comp* ctx, int distance, int len);
```

Comment:
These style/format changes should be in their own commit; that way, it is easy to identify which "functional" changes are actually required to make this work without all the noises.